benchmarks/Dapper.Tests.Performance/Benchmarks.RepoDB.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System.ComponentModel;
 using System.Linq;
 using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
 using RepoDb;
 
 namespace Dapper.Tests.Performance
 {
+#if !NET5_0_OR_GREATER

```

Comment:
can we get a `// something` comment here (doesn't have to be huge) re why we exclude this? can be as simple as "fails with {SomeException}" - just a hint